File: 2. Count number of vowels, consonants, spaces in String/code.cpp
#include <iostream>
#include <string>
using namespace std;

void countCharacters(string s, int& vowels, int& consonants, int& spaces) {
    vowels = 0;
    consonants = 0;
    spaces = 0;
    for (char c : s) {
        if (tolower(c) == 'a' || tolower(c) == 'e' || tolower(c) == 'i' || tolower(c) == 'o' || tolower(c) == 'u') {
            vowels++;
        } else if (isalpha(c)) {
            consonants++;
        } else if (isspace(c)) {
            spaces++;
        }
    }
}

int main() {
    string input_str;
    cout << "Enter a string: ";
    getline(cin, input_str);
    int vowels, consonants, spaces;
    countCharacters(input_str, vowels, consonants, spaces);
    cout << "Vowels: " << vowels << endl;
    cout << "Consonants: " << consonants << endl;
    cout << "White spaces: " << spaces << endl;
    return 0;
}

File: 2. Count number of vowels, consonants, spaces in String/code.py
def count_characters(s):
    vowels = 0
    consonants = 0
    spaces = 0
    for char in s:
        if char.lower() in 'aeiou':
            vowels += 1
        elif char.isalpha():
            consonants += 1
        elif char.isspace():
            spaces += 1
    return vowels, consonants, spaces

input_str = input("Enter a string: ")
vowels, consonants, spaces = count_characters(input_str)
print("Vowels:", vowels)
print("Consonants:", consonants)
print("White spaces:", spaces)
